pkg/scalers/datadog_scaler.go

```diff
@@ -37,6 +38,9 @@ type datadogMetadata struct {
 	fillValue            float64
 }
 
+const maxString = "max"
+const avgString = "average"
```

Comment:
Even though this is not Datadog code, I would still keep Datadog convention of "avg" for average

Reply:
I had wondered this, but wanted to keep consistency with the other params being defined; like "MetricType" uses Average/Global.
If you feel strongly about it, I will change it?